javascript/schema.js

```diff
@@ -1,5 +1,6 @@
 export const defaultSchema = {
   reflexAttribute: 'data-reflex',
   reflexPermanentAttribute: 'data-reflex-permanent',
-  reflexRootAttribute: 'data-reflex-root'
+  reflexRootAttribute: 'data-reflex-root',
+  reflexInheritAttribute: 'data-reflex-inherit'
```

Comment:
I propose a rename of this new attribute given that all other attributes are noun based descriptors rather than verbs indicating behavior. 

`data-reflex-inherit` -> `data-reflex-dataset=combined`

Reply:
Updated it to use `data-reflex-dataset=combined` 🙌